olioapps/typescript-starter-lab
# Write functions and pass all tests to solve all problems

# What
* Add a functions file containing necessary util functions to solve Animal Fight problems. Write tests in Index file and pass all tests.

![Animal Fight PR](https://user-images.githubusercontent.com/49892118/73107484-c6e1a980-3eb2-11ea-97fe-3dccf69ba495.png)

# How To Test
* Use terminal to:
1.  Clone Repo.
2. Npm or yarn install all packages.
3. Npm run dev.
*Terminal should display all passing messages for tests.
